examples/web-components/first-time-orientation/src/first-time-orientation.scss

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+$css--plex: true !default;
+@use '@carbon/styles';
+@use '@carbon/styles/scss/theme' as *;
+@use '@carbon/styles/scss/spacing' as *;
+@use '@carbon/styles/scss/type' as *;
+@use '@carbon/type/scss/font-family' as *;
+@use '@carbon/styles/scss/components/tag' as *;
+
+$prefix: 'clabs' !default;
+
+.#{$prefix}__wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  block-size: 100vh;
+}
+.#{$prefix}__flexContainer {
```

Comment:
are we using camel-case in CLABs? usually as a convention we use hypens for separation `flex-container`